### PR TITLE
codec(cdc) Removing logging of message bodies to avoid PII issues

### LINKED ISSUE #9031: Row content being logged as part of CDC ErrMessageTooLarge logging
### What did you do?

If a message is too large, both the Canal JSON and the Open Protocol encoders are logging the full message.

You can see the issue in 3 files:
canal_json_row_event_encoder.go:347
canal_json_tx_event_encoder.go:64
canal_json_row_event_encoder.go:69

### What did you expect to see?

The message itself (the row data) should not be printed in logs as this is a PII violation

### What did you see instead?

The content of the row

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
(paste TiDB cluster version here)
```

Upstream TiKV version (execute `tikv-server --version`):

```console
(paste TiKV version here)
```

TiCDC version (execute `cdc version`):

```console
v7.0.0
```

### PR DESCRIPTION
**Pull request by darraes**
### What problem does this PR solve?
Issue is described here https://github.com/pingcap/tiflow/issues/9031

Issue Number: close #9031

### What is changed and how it works?
We simply print the length of the value instead of its contents

### Check List 
make test succeeded

#### Tests 

 - Unit test

### Release note 
`None`.

### CODE REVIEW

**Comment by charleszheng44** [comment]
@darraes would you sign the [Contributor License Agreement](https://cla.pingcap.net/pingcap/tiflow?pullRequest=9032)

**Review by charleszheng44 — APPROVED** [review]

**Comment by darraes** [comment]
@charleszheng44 I signed it. One thing I can't figure out is that I did add the None for the release note but the do-not-merge/release-note-label-needed label is there.

Any ideas?

**Comment by charleszheng44** [comment]
> @charleszheng44 I signed it. One thing I can't figure out is that I did add the None for the release note but the do-not-merge/release-note-label-needed label is there.
> 
> Any ideas?

Could you try to replace the section with

```release-note
`None`.
```

File: pkg/sink/codec/canal/canal_json_row_event_encoder.go
Comment (nongfushanquan) [inline pkg/sink/codec/canal/canal_json_row_event_encoder.go:351]: There  is already  a length value above.

Comment (charleszheng44) [reply pkg/sink/codec/canal/canal_json_row_event_encoder.go:351]: :+1:

Comment (darraes) [reply pkg/sink/codec/canal/canal_json_row_event_encoder.go:351]: @nongfushanquan Even the PK might contain PII. I removed the duplicated length logging. Didn't realize it was already there.

**Comment by darraes** [comment]
@charleszheng44 @nongfushanquan Thanks for the review. 

I still don't know what the check-issue-triage-complete wants... I see the missing release note label is still there even though I added the note. The message now says it found may-affects on the bug.

Docs say "_For type/bug issue with severity/critical or severity/major label, the checker checks if there is any may-affects-x.y label, which means the issue has not been diagnosed on all needed versions. If there is, the pull request is blocked and not able to be merged. So in order to merge a bugfix pull request into the target branch, every other effective version needs to first be diagnosed. TiDB maintainer will add these labels._". I don't understand what is the action on my side from this message.

How do I work around this?

**Review by nongfushanquan — APPROVED** [review]

**Comment by darraes** [comment]
The tests were failing so I updated the PR by rebasing it on top of master. But now I lost the label.
@nongfushanquan @asddongmen I think I need a new approval.
Sorry about that.

**Comment by darraes** [comment]
@nongfushanquan I signed the CLA. Not sure why the check didn't clear. The jenkins/verify, anything I need to do there?

**Comment by asddongmen** [comment]
> Daniel Arraes seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

@darraes Thank you for your contribution! The prompt here says that you may not have added the email address you used in this commit to the email list on your Github account, so the CLA check did not pass. Could you please check the email list on your Github account?

**Comment by darraes** [comment]
@asddongmen The commit went into with my Airbnb email. I added that to the account (see attached picture). However when I click on the cla link it says it is already signed even though it is not cleared in the check.

 
Screenshot 2023-05-31 at 10 01 03 AM

**Pull request merged**
